docs/site/test-runner.md

```diff
@@ -24,6 +24,14 @@ Toggle between implementation and test | - | Switches the file between implement
 
 You can enable the Calva setting "Test on Save" to have tests for the current namespace run on file save.
 
+## Test Explorer
+
+Calva has experimental support for showing test results in the VSCode Test Explorer UI. You can enable this support by setting `calva.useTestExplorer` to `true`. When you enable this setting, the Testing icon will appear in the VSCode Activilty Bar.
+
+With this feature you will be able to browse and run tests directly from the Testing tab.
```

Comment:
I think we should add something about what the Test Explorer is. Or maybe just link Test Explorer UI to its marketplace page.

Reply:
Shall we get this merged and pull the docs changes out into a separate PR?

Reply:
```suggestion
Calva has experimental support for showing test results in the [VSCode Test Explorer UI](https://marketplace.visualstudio.com/items?itemName=hbenl.vscode-test-explorer). You can enable this support by setting `calva.useTestExplorer` to `true`. When you enable this setting, the Testing icon will appear in the VSCode Activilty Bar.

With this feature (and the TestExplorer UI extension installed) you will be able to browse and run tests directly from the Testing tab.
```